nncf/quantization/advanced_parameters.py

```diff
@@ -551,13 +551,15 @@ def apply_advanced_parameters_to_config(
         initializer["batchnorm_adaptation"] = {"num_bn_adaptation_samples": 0}
         config["initializer"] = initializer
 
-    activations_config = convert_quantization_parameters_to_dict(params.activations_quantization_params)
-    if activations_config:
-        config["activations"] = activations_config
+    if isinstance(params.activations_quantization_params, QuantizationParameters):
```

Comment:
@alexsu52 is it valid condition?
quantization_params got annotation `QuantizationParameters | FP8QuantizationParameters | None` but convert_quantization_parameters_to_dict expects only `QuantizationParameters | None`.
It's looks like used only for TF, so it does not support FP8 quantization.
May be need add extra check before and raise exception?

Reply:
Yes, you are right.